radar.lua

```diff
@@ -121,7 +121,6 @@ s_protect.command_radar = function(name)
 		"label[7,4.25;" .. S("Access for everybody") .. "]" ..
 		"image[6,5;1,1;simple_protection_radar_down.png]" ..
 		"image[7,5;1,1;simple_protection_radar_up.png]" ..
-		"label[6,6;One area unit ("..s_protect.claim_height
-			.. "m) up/down\n= no claims on this Y level]"
+		"label[6,6;" .. S("One area unit (@1m) up/down\n-> no claims on this Y level", s_protect.claim_height) .. "]"
```

Comment:
Line too long. Please break it after `",`

Reply:
line break : ok